pom.xml

```diff
@@ -115,7 +115,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <java.version>11</java.version>
+    <java.version>1.8</java.version>
```

Comment:
Thanks!

Reply:
Let's wait for the fix of PySpark and SparkR?